scopes/envs/envs/environment.ts

```diff
@@ -1,4 +1,21 @@
 import { Linter } from '@teambit/linter';
+import { Tester } from '@teambit/tester';
+import { Bundler, BundlerContext, DevServer, DevServerContext } from '@teambit/bundler';
+import { BuildTask } from '@teambit/builder';
+import { SchemaExtractor } from '@teambit/schema';
+import { WebpackConfigTransformer } from '@teambit/webpack';
+import { PackageJsonProps } from '@teambit/pkg';
+import { TsConfigSourceFile } from 'typescript';
+
+export type EnvDescriptor = {
+  type: string;
+};
+
+export interface PkgDependencies {
```

Comment:
This type shouldn't be here. should probably be in the pkg or in dependency resolver.